lntest/itest/lnd_test.go

```diff
@@ -10172,23 +10172,19 @@ func testMultiHopHtlcLocalTimeout(net *lntest.NetworkHarness, t *harnessTest) {
 		t.Fatalf("htlc mismatch: %v", predErr)
 	}
 
-	// We'll mine defaultCSV blocks in order to generate the sweep
-	// transaction of Bob's funding output. This will also bring us to the
-	// maturity height of the htlc tx output.
-	if _, err := net.Miner.Node.Generate(defaultCSV); err != nil {
-		t.Fatalf("unable to generate blocks: %v", err)
-	}
-
+	// With the closing transaction confirmed, we should expect our HTLC
+	// timeout transaction to be broadcast.
```

Comment:
our -> Bob's. Add here that because the outgoing broadcast delta is zero, there is no cltv wait required?

Reply:
I mean that it could be that after the closing tx confirms, the resolver still needs to wait for the cltv lock. But not in this case, because the delta is zero. So channel arbitrator waits with publishing the commit tx until the exact block the htlc expires?

Reply:
Correct. What I meant is that the expiry is at height 100, so we decide to close then due to a 0 outgoing broadcast delta, but the timeout is actually being broadcast at height 101 because we wait until the confirmation of the closing transaction.

Reply:
Ok